test/TestProjects/BodyAndPath-LowLevel/Generated/BodyAndPathClient.cs

```diff
@@ -138,20 +138,20 @@ public virtual Response CreateStream(string itemNameStream, RequestContent conte
             }
         }
 
-        /// <summary> Resets products. </summary>
+        /// <summary> List products by the name. </summary>
         /// <param name="enumName1"> The first name. Allowed values: &quot;current&quot; | &quot;default&quot;. </param>
         /// <param name="enumName2"> The second name. Allowed values: &quot;latest&quot;. </param>
         /// <param name="content"> The content to send as the body of the request. </param>
         /// <param name="options"> The request options. </param>
 #pragma warning disable AZC0002
-        public virtual async Task<Response> CreateEnumAsync(string enumName1, string enumName2, RequestContent content, RequestOptions options = null)
+        public virtual async Task<Response> GetAllByNameAsync(string enumName1, string enumName2, RequestContent content, RequestOptions options = null)
```

Comment:
Can we cover all the test cases for List -> Get, List<resource_name_singular> -> Get<resource_name_plural>..etc?